data/tlg0007/tlg068/tlg0007.tlg068.perseus-eng3.xml

```diff
@@ -0,0 +1,1756 @@
+         		<name>Lisa Cerrato</name>
+         		<name>Rashmi Singhal</name>
+         		<name>Bridget Almas</name>
+         	</respStmt>
+         	<funder n="org:NEH">The National Endowment for the Humanities</funder>
+        </titleStmt>
+         <!--<extent/>&Perseus.publish;-->
+      	<publicationStmt>
+      		<publisher>Trustees of Tufts University</publisher>
+      		<pubPlace>Medford, MA</pubPlace>
+      		<authority>Perseus Project</authority>
+      		<date type="release">2010-12-13'</date>
+      	</publicationStmt>
+      	<sourceDesc>
+            <biblStruct>
+               <analytic/>
+               <monogr>
+               	<author>Plutarch</author>
+                  <title xml:lang="lat">Moralia</title>
+                  <respStmt>
+                     <resp>with an English Translation by</resp>
+                     <name>Frank Cole Babbitt</name>
+                  </respStmt>
+                  <imprint>
+                     <pubPlace>Cambridge, MA</pubPlace>
+                     <publisher>Harvard University Press</publisher>
+                     <pubPlace>London</pubPlace>
+                     <publisher>William Heinemann Ltd.</publisher>
+                     <date>1927</date>
+                  </imprint>
+                  <biblScope unit="volume">1</biblScope>
+               </monogr>
+            	<ref target="https://archive.org/details/in.ernet.dli.2015.43837/page/n117">The Internet Archive</ref>
+            </biblStruct>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <!--<editorialDecl>
+          <correction status="high" method="silent">
+               <p>optical character recognition</p>
+            </correction>
+         </editorialDecl>-->
+      	<p>Text encoded in accordance with the latest EpiDoc standards</p> 
+      	<p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE Architecture</p>    
+   <refsDecl n="CTS">
+   	<cRefPattern n= "section" matchPattern="(\w+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+   		<p>This pointer pattern extracts sections</p></cRefPattern>
+            <!--<state unit="section"/>-->
+         </refsDecl>
+      	<!--<refsDecl doctype="TEI.2">
+				<state unit="stephpage" n="chunk"/>
+				</refsDecl>-->
+      </encodingDesc>
+      <profileDesc>
+         <langUsage>
+            <language ident="eng">English</language>
+            <language ident="grc">Greek</language>
+         	<language ident="lat">Latin</language>
+         	<language ident="deu">German</language>
+         </langUsage>
+      </profileDesc>
+      <revisionDesc>
+         <change who="RS" when="2010-05">tagged and parsed</change>
+      	<change who="CHS 2019 Interns" when="2019-06-06">CTS and Epidoc conversion</change>
+      </revisionDesc>
+   </teiHeader>
+   <text>
+      <body>
+      	<div type="translation" xml:lang="eng" n="urn:cts:greekLit:tlg007.tlg068.perseus-eng3">
```

Comment:
The tests are failing because the file name is here is wrong.
This should be tlg0007.tlg068.perseus-eng3 (a zero is missing)

Reply:
In general, we like to change anything that is a formatting tag such as `<hi rend="italics">` to a tag that illustrates **why** it is in italics.
This cannot always be done easily in a whole file and can take time, but here I am seeing things like _ibid_ or _cf._ in italics.

There are various ways of dealing with this. Ibid and cf. are in italics because they are Latin. So I generally use `<foreign xml:lang="lat">` (Other may just use an `<emph>` tag since they are Latin words that represent bibliographic abbreviations particular to English footnote.)

The other things I spotted are `<hi rend="italics"><title>`. This should be `<title>` or `<title rend="italic">`.  (note that the TEI prefers "italic" to "italics". In general, older Perseus files let the software italicize the titles, but the latter is a better tag.

There may be other items here that are italicized for other reasons. The other reason is usually just emphasis so `<emph rend="italic">

In a case like `<hi rend="italics">Ibid., Adesp.</hi>` this is wrong. Ibid. is italics because it is Latin and Adesp. is a title. So `<foreign xml:lang="lat">Ibid.</foreign>, <title rend="italic">Adesp.</title>`

Reply:
This is not necessarily a conversion issue and not an error, but something to keep in mind.

Another odd thing in this file I see is that there is a space before some punctuation. 
`...  whom you do not deceive ?`
`... in their desire to be alive ;`
It would be good to correct things like this if they catch your eye. These are bad OCR remnants. If it doesn't look like it should look, it may well be wrong.

Reply:
@lcerrato A quick question before I resubmit the file- the spaces before punctuation exist in the scan of the text. Would you rather I preserve what's in the scan or edit punctuation so the spacing is more correct?

Reply:
They may appear to be spaces, but that is just a nasty artifact of the old printing style. We don't want them there. They will break other text mining tools.

Reply:
Also, this is not a deal breaker. I wouldn't worry about catching them all. Just look for ? ;  : (for instance). 
Now that I see them here, I'll  do a global sweep across the whole directory myself once everything is checked in. I am sure there are plenty of these lurking in all of the files. These sorts of issues aren't typically things I would look for and it just caught my eye.